universal-login-sdk/test/e2e/DeployedWallet.ts

```diff
@@ -62,4 +62,31 @@ describe('E2E: DeployedWallet', async () => {
       expect(connectedDevices.map(({publicKey}: any) => publicKey)).to.include(address);
     }).timeout(15000);
   });
+
+  describe('getBalance', async () => {
+    const initialEthBalance = '1999999999999316000';
+    const initialMockTokenBalance = '1000000000000000000';
+
+    it('initial eth balance', async () => {
+      const balance = await deployedWallet.getBalance(ETHER_NATIVE_TOKEN.address);
+      expect(balance).to.equal(initialEthBalance);
+    });
+
+    it('initialEthBalance + 1 ETH', async () => {
+      await otherWallet.sendTransaction({to: deployedWallet.contractAddress, value: constants.WeiPerEther});
+      const balance = await deployedWallet.getBalance(ETHER_NATIVE_TOKEN.address);
+      expect(balance).to.equal(constants.WeiPerEther.add(initialEthBalance));
+    });
+
+    it('initial mockToken balance', async () => {
+      const balance = await deployedWallet.getBalance(mockToken.address);
+      expect(balance).to.equal(initialMockTokenBalance);
+    });
+
+    it('initial mockToken balance + sent value', async () => {
+      await mockToken.transfer(deployedWallet.contractAddress, constants.WeiPerEther);
+      const balance = await deployedWallet.getBalance(mockToken.address);
+      expect(balance).to.equal(constants.WeiPerEther.add(initialMockTokenBalance));
```

Comment:
I would just make it more explicit: `initialMockTokenBalance.add(constants.WeiPerEther)`